sdv/sequential/par.py

```diff
@@ -45,21 +54,37 @@ class PARSynthesizer:
             Whether to print progress to console or not.
     """
 
+    _model_sdtype_transformers = {
+        'categorical': None,
+        'numerical': None,
+        'boolean': None
+    }
+
     def _get_context_metadata(self):
         context_columns_dict = {}
-        context_columns = self.context_columns or []
+        context_columns = self.context_columns.copy() if self.context_columns else []
+        if self._sequence_key:
+            context_columns += self._sequence_key
+
         for column in context_columns:
             context_columns_dict[column] = self.metadata._columns[column]
 
         context_metadata_dict = {'columns': context_columns_dict}
         return SingleTableMetadata._load_from_dict(context_metadata_dict)
 
-    def __init__(self, metadata, enforce_min_max_values, enforce_rounding, context_columns=None,
-                 segment_size=None, epochs=128, sample_size=1, cuda=True, verbose=False):
-        self.metadata = metadata
+    def __init__(self, metadata, enforce_min_max_values=True, enforce_rounding=False,
+                 context_columns=None, segment_size=None, epochs=128, sample_size=1, cuda=True,
+                 verbose=False):
+        super().__init__(
+            metadata=metadata,
+            enforce_min_max_values=enforce_min_max_values,
+            enforce_rounding=enforce_rounding,
+        )
+        sequence_key = self.metadata._sequence_key
+        self._sequence_key = list(cast_to_iterable(sequence_key)) if sequence_key else None
```

Comment:
cast_to_iterable already returns a list

Reply:
`sequence_key` can be a tuple and if it is then `cast_to_iterable` returns a tuple